docs/conceptual/runtime/weight.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Transaction Weight
+---
+
+A number of resources in any chain can be limited by definition, such as time, memory and
+computation. A mechanism should exist to prevent individual resource-consuming components of the
+chain, including but not limited to dispatchable functions, from consume too much of any of the
+mentioned resources. This concept is represented in substrate runtime by weights. Furthermore, consuming
+some weights could optionally incur some fee.
+
+The fee implications of the weight system is covered in the [Fee Developer document](). This
+document mostly covers the concept of weights.
+
+## Transaction Weight
+
+As mentioned, weights can technically refer to, or represent, numerous _limited_ resources. A custom
```

Comment:
```suggestion
Weights represent the _limited_ resources of your blockchain, for example computational cycles, memory, storage, etc... A custom
```

I dunno how else to say it, but the language throughout is a little verbose and cryptic when I think it could be more direct and solid.

Like rather than "can technically refer to, or represent", just say it represents. Anyway, this is not something that I expect you to clean up, just want to hear if you have feedback about this.